tests/README.md

```diff
@@ -1,18 +1,52 @@
-To run test, execute from project root:
+## Test Structure
 
+Our tests are organized into several subdirectories, each focusing on different aspects of our application:
+
+* api: Contains tests related to the MindsDB's API endpoints.
+* integration_tests: Contains the integration tests
+* load: Contains the load tests
+* scripts: Scripts and utilitis used for tests
+* unit: This directory contains the unit tests:
+        * handlers: A subset of unit tests specifically targeting data handlers.
+        * ml_handlers: A subset of unit tests specifically targeting ML handlers.
+
+## Installation
+
+To run the tests, you need to install the necessary dependencies. You can do this by running the following command:
+
+```
+pip install -r requirements/requirements-test.txt && pip install -r  requirements/requirements.txt
```

Comment:
Even with this, if someone were to run `pytest -vx unit` (or on a specific sub-directory within it), wouldn't it fail because handler specific dependencies are still not installed?

Reply:
Good point, I will update the doc